dashboard/app/assets/stylesheets/application.scss

```diff
@@ -4076,13 +4156,15 @@ html[dir=rtl] {
 
     img {
       // paint native Microsoft icon full black and set it's color to $neutral_white
-      filter: brightness(0) saturate(100%) invert(100%) sepia(0%) saturate(0%) hue-rotate(111deg) brightness(101%) contrast(100%);
+      filter: brightness(0) saturate(100%) invert(100%) sepia(0%) saturate(0%)
+        hue-rotate(111deg) brightness(101%) contrast(100%);
     }
   }
 
   img {
     // paint native Microsoft icon full black and set it's color to $neutral_dark
-    filter: brightness(0) saturate(100%) invert(15%) sepia(31%) saturate(359%) hue-rotate(169deg) brightness(94%) contrast(91%);
+    filter: brightness(0) saturate(100%) invert(15%) sepia(31%) saturate(359%)
```

Comment:
I don't think we should add these auto-formatting changes in this PR, I would be worried that they would change something else and it also hides where the actual change is made.